jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/deploy/DeployServerlessApplicationDialog.kt

```diff
@@ -121,6 +128,18 @@ class DeployServerlessApplicationDialog(
             view.stacks.selected()
         } ?: throw RuntimeException(message("serverless.application.deploy.validation.stack.missing"))
 
+    val stackId: String?
+        get() = if (view.createStack.isSelected) {
+            null
+        } else {
+            val stackName = view.stacks.selected()
```

Comment:
stackName?.let {
	stackNameToStackId[it]
}

Reply:
Nice! I thought that I could use something concise like you suggested, it just didn't click to my mind at that time.